test/inductor/test_compile_subprocess.py

```diff
@@ -75,6 +83,78 @@ def tearDown(self):
         TestCase.tearDown(self)
         torch._dynamo.reset()
 
+    @requires_gpu()
+    @requires_triton()
+    @unittest.skipIf(
+        not IS_BIG_GPU, "Skipping triton backend only since not big GPU (not enough SM)"
+    )
+    def test_progressive(self):
+        from triton.testing import do_bench
+
+        from torch._inductor.compile_fx_async import _ProgressiveFxCompile
+
+        torch._inductor.compile_fx.fx_compile_progressive = True
+
+        x = torch.randn(100, 100, device=GPU_TYPE)
+        y = torch.randn(100, 100, device=GPU_TYPE)
+
+        @torch.compile(fullgraph=True, backend="inductor")
+        def matmul(x, y):
+            return x @ y
+
+        _ProgressiveFxCompile._reset_stats()
+
+        with contextlib.ExitStack() as stack:
+            # TODO: make caches work with progressive compile
```

Comment:
Makes me wonder if we should have something like torch._inductor.compile_fx_async.BUG_CACHES_DONT_WORK_WITH_ASYNC that we check in the tests so when we fix it they all start failing and we remember to fix them, lol